libimage/runtime_test.go

```diff
@@ -94,3 +103,43 @@ func TestTmpdir(t *testing.T) {
 		os.Unsetenv(tmpStr)
 	}
 }
+
+func TestRuntimeListImagesAllImages(t *testing.T) {
+	runtime := testNewRuntime(t)
```

Comment:
(Note to self: I didn’t check how the test infrastructure / isolation from the surrounding environment works.)

Reply:
Yes, this looks good — or, at least, it is used in three dozen other places, so it better be good :)